digital-credentials/support/helper.js

```diff
@@ -29,21 +33,52 @@ export function makeGetOptions(requestsToUse, mediation = "required") {
         requests.push(makeOID4VPDict());
         break;
       case "default":
-        requests.push(makeDigitalCredentialRequest(undefined, undefined));
+        requests.push(makeDigitalCredentialGetRequest(undefined, undefined));
         break;
       default:
         throw new Error(`Unknown request type: ${request}`);
     }
   }
   return { digital: { requests }, mediation };
 }
+
+/**
+ * @param {CreateProtocol | CreateProtocol[]} [requestsToUse=["default"]]
+ * @param {CredentialMediationRequirement} [mediation="required"]
+ * @returns {CredentialCreationOptions}
+ */
+export function makeCreateOptions(requestsToUse, mediation = "required") {
+  if (typeof requestsToUse === "string") {
+    if (requestsToUse === "default" || requestsToUse === "openid4vci") {
+      return makeCreateOptions([requestsToUse], mediation);
+    }
+  }
+  if (!Array.isArray(requestsToUse) || !requestsToUse?.length) {
+    return { digital: { requests: requestsToUse }, mediation };
+  }
+  const requests = [];
+  for (const request of requestsToUse) {
+    switch (request) {
+      case "openid4vci":
+        requests.push(makeOID4VCIDict());
+        break;
+      case "default":
+        requests.push(makeDigitalCredentialCreateRequest(undefined, undefined));
```

Comment:
This should be the same as... but if you want to do it explicitly, that's ok too. 

```suggestion
        requests.push(makeDigitalCredentialCreateRequest());
```